### PR TITLE
ROPC remediation - more files with connection strings

### LINKED ISSUE #33991: ROPC remediation: more files with connection strings
Remediate remaining files that are flagged for having connection strings, except for one Blazor file.

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #33991 

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/security/app-secrets.md](https://github.com/dotnet/AspNetCore.Docs/blob/88eb2bb3d0daf1f0d8ce3837251e42c88c71b3ee/aspnetcore/security/app-secrets.md) | [Safe storage of app secrets in development in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/security/app-secrets?branch=pr-en-us-33990) |
| [aspnetcore/security/data-protection/configuration/overview.md](https://github.com/dotnet/AspNetCore.Docs/blob/88eb2bb3d0daf1f0d8ce3837251e42c88c71b3ee/aspnetcore/security/data-protection/configuration/overview.md) | [Configure ASP.NET Core Data Protection](https://review.learn.microsoft.com/en-us/aspnet/core/security/data-protection/configuration/overview?branch=pr-en-us-33990) |
| [aspnetcore/signalr/redis-backplane.md](https://github.com/dotnet/AspNetCore.Docs/blob/88eb2bb3d0daf1f0d8ce3837251e42c88c71b3ee/aspnetcore/signalr/redis-backplane.md) | [Set up a Redis backplane for ASP.NET Core SignalR scale-out](https://review.learn.microsoft.com/en-us/aspnet/core/signalr/redis-backplane?branch=pr-en-us-33990) |

### CODE REVIEW

File: aspnetcore/performance/caching/distributed/samples/8.x/RedisCache/appsettings.json
Comment (tdykstra) [inline aspnetcore/performance/caching/distributed/samples/8.x/RedisCache/appsettings.json:1]: Including password here isn't required, as it and the other parts of the connection string were there just to give an impression of what an Azure Redis Cache connection string looks like.

File: aspnetcore/security/app-secrets.md [hunk 1]
```diff
@@ -201,21 +202,13 @@ The `Movies:ConnectionString` and `Movies:ServiceApiKey` secrets are mapped to t
...
-Storing passwords in plain text is insecure. For example, a database connection string stored in `appsettings.json` may include a password for the specified user:
-
-[!code-json[](~/security/app-secrets/samples/3.x/UserSecrets/appsettings-unsecure.json?highlight=3)]
-
-A more secure approach is to store the password as a secret. For example:
+Storing passwords in plain text is insecure. For example, a database connection string stored in `appsettings.json` should not include a password. Instead, store the password as a secret, and include the password in the connection string at runtime. For example:
 
 ```dotnetcli
-dotnet user-secrets set "DbPassword" "pass123"
+dotnet user-secrets set "DbPassword" "`<secret value>`"
```
Comment (Rick-Anderson) [inline aspnetcore/security/app-secrets.md:208]: For bonus points, replace "`<secret value>`"
with
`"\x00``<secret value>``\"`  which should throw an error, although maybe `<secret value>` will throw. We don't want them copy/paste something that works.

File: aspnetcore/security/app-secrets/includes/app-secrets-3-5.md [hunk 2]
```diff
@@ -172,23 +173,15 @@ The `Movies:ConnectionString` and `Movies:ServiceApiKey` secrets are mapped to t
 
 ## String replacement with secrets
 
-Storing passwords in plain text is insecure. For example, a database connection string stored in `appsettings.json` may include a password for the specified user:
-
-[!code-json[](~/security/app-secrets/samples/3.x/UserSecrets/appsettings-unsecure.json?highlight=3)]
-
-A more secure approach is to store the password as a secret. For example:
+Storing passwords in plain text is insecure. For example, a database connection string stored in `appsettings.json` should not include a password. Instead, store the password as a secret, and include the password in the connection string at runtime. For example:
```
Comment (Rick-Anderson) [inline aspnetcore/security/app-secrets/includes/app-secrets-3-5.md:176]: do you want to tell them never store secrets in a config file (for example `appsettings.json`) ... checked in source.

**Review by Rick-Anderson — APPROVED** [review]

File: aspnetcore/security/app-secrets.md (hunk 1 above)
Comment (tdykstra) [reply aspnetcore/security/app-secrets.md:208]: If someone is determined to use it as an actual password, I'm afraid we would only be giving them a particularly strong one.

File: aspnetcore/security/app-secrets/includes/app-secrets-3-5.md (hunk 2 above)
Comment (tdykstra) [inline aspnetcore/security/app-secrets/includes/app-secrets-3-5.md:176]:
```suggestion
Storing passwords in plain text is insecure. Never store secrets in a configuration file such as `appsettings.json`, which might get checked in to a source code repository.

For example, a database connection string stored in `appsettings.json` should not include a password. Instead, store the password as a secret, and include the password in the connection string at runtime. For example:
```

File: aspnetcore/security/app-secrets.md [hunk 3]
```diff
@@ -201,21 +202,13 @@ The `Movies:ConnectionString` and `Movies:ServiceApiKey` secrets are mapped to t
 
 ## String replacement with secrets
 
-Storing passwords in plain text is insecure. For example, a database connection string stored in `appsettings.json` may include a password for the specified user:
-
-[!code-json[](~/security/app-secrets/samples/3.x/UserSecrets/appsettings-unsecure.json?highlight=3)]
-
-A more secure approach is to store the password as a secret. For example:
+Storing passwords in plain text is insecure. For example, a database connection string stored in `appsettings.json` should not include a password. Instead, store the password as a secret, and include the password in the connection string at runtime. For example:
```
Comment (tdykstra) [inline aspnetcore/security/app-secrets.md:205]:
```suggestion
Storing passwords in plain text is insecure. Never store secrets in a configuration file such as `appsettings.json`, which might get checked in to a source code repository.

For example, a database connection string stored in `appsettings.json` should not include a password. Instead, store the password as a secret, and include the password in the connection string at runtime. For example:
```

**Pull request merged**